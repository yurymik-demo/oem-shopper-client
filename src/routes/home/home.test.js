import React from "react";
import HomeView from "./home";
import renderer from "react-test-renderer";

jest.mock("../../components/ItemsList/ItemsList", () => "ItemsListComponent");

it("renders without crashing", () => {
  const wrapper = renderer.create(<HomeView />).toJSON();
  expect(wrapper).toMatchSnapshot();
});
