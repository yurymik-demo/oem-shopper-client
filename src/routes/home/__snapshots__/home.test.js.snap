// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders without crashing 1`] = `
<div>
  <h1>
    Best Deals
  </h1>
  <ItemsListComponent />
</div>
`;
